tests/data/flamegraph/perf-vertx-stacks/perf-vertx-stacks-01-collapsed-all.svg

```diff
@@ -1,1844 +0,0 @@
-<?xml version="1.0" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg version="1.1" width="1200" height="1190" onload="init(evt)" viewBox="0 0 1200 1190" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
It looks like the contents of this file got deleted, which is causing some of the tests to fail.

Reply:
Should be fixed now.